server/venueless/core/models/auth.py

```diff
@@ -18,4 +26,16 @@ class Meta:
     def serialize_public(self):
         # Important: If this is updated, venueless.core.services.user.get_public_users also needs to be updated!
         # For performance reasons, it does not use this method directly.
-        return {"id": str(self.id), "profile": self.profile}
+        return {
+            "id": str(self.id),
+            "profile": self.profile,
+            "moderation_state": self.moderation_state,
```

Comment:
Is this really public? I don't think any user needs to know if an arbitrary other user is silenced, feels like a data leak